CLI example from guide do not work on PowerShell on Windows

### Describe the bug

On the guide for the [scheduler](https://quarkus.io/guides/scheduler), the command to create the example project is :
> quarkus create app org.acme:scheduler-quickstart --extension=resteasy-reactive,scheduler --no-code

### Expected behavior

The example scheduler-quickstart project should be created.

### Actual behavior

However on PowerShell, it returns :
> [ERROR] Unable to create project: Unmatched argument at index 4: 'resteasy-reactive'

Adding quotes avoids this error :
> quarkus create app org.acme:scheduler-quickstart --extension='resteasy-reactive,scheduler' --no-code

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

--- a/docs/src/main/asciidoc/includes/devtools/create-app.adoc
+++ b/docs/src/main/asciidoc/includes/devtools/create-app.adoc
@@ -31,10 +31,10 @@ endif::[]
 endif::[]
 ifdef::create-app-extensions[]
 ifndef::create-app-code[]
-    --extension={create-app-extensions} \
+    --extension='{create-app-extensions}' \
 endif::[]
 ifdef::create-app-code[]
-    --extension={create-app-extensions}
+    --extension='{create-app-extensions}'
 endif::[]
 endif::[]
 ifndef::create-app-code[]
@@ -82,10 +82,10 @@ endif::[]
 endif::[]
 ifdef::create-app-extensions[]
 ifndef::create-app-code[]
-    -Dextensions="{create-app-extensions}" \
+    -Dextensions='{create-app-extensions}' \
 endif::[]
 ifdef::create-app-code[]
-    -Dextensions="{create-app-extensions}"
+    -Dextensions='{create-app-extensions}'
 endif::[]
 endif::[]
 ifndef::create-app-code[]

--- a/docs/src/main/asciidoc/includes/devtools/create-cli.adoc
+++ b/docs/src/main/asciidoc/includes/devtools/create-cli.adoc
@@ -31,10 +31,10 @@ endif::[]
 endif::[]
 ifdef::create-cli-extensions[]
 ifndef::create-cli-code[]
-    --extension={create-cli-extensions} \
+    --extension='{create-cli-extensions}' \
 endif::[]
 ifdef::create-cli-code[]
-    --extension={create-cli-extensions}
+    --extension='{create-cli-extensions}'
 endif::[]
 endif::[]
 ifndef::create-cli-code[]
@@ -72,10 +72,10 @@ ifndef::create-cli-code[]
     -DnoCode \
 endif::[]
 ifdef::create-cli-extensions[]
-    -Dextensions="picocli,{create-cli-extensions}"
+    -Dextensions='picocli,{create-cli-extensions}'
 endif::[]
 ifndef::create-cli-extensions[]
-    -Dextensions="picocli"
+    -Dextensions='picocli'
 endif::[]
 ifdef::create-cli-post-command[]
 {create-cli-post-command}

--- a/docs/src/main/asciidoc/includes/devtools/extension-add.adoc
+++ b/docs/src/main/asciidoc/includes/devtools/extension-add.adoc
@@ -8,7 +8,7 @@ ifdef::devtools-wrapped[+]
 [source,bash,subs=attributes+,role="secondary asciidoc-tabs-sync-maven"]
 .Maven
 ----
-./mvnw quarkus:add-extension -Dextensions="{add-extension-extensions}"
+./mvnw quarkus:add-extension -Dextensions='{add-extension-extensions}'
 ----
 endif::[]
 ifndef::devtools-no-gradle[]
@@ -16,6 +16,6 @@ ifdef::devtools-wrapped[+]
 [source,bash,subs=attributes+,role="secondary asciidoc-tabs-sync-gradle"]
 .Gradle
 ----
-./gradlew addExtension --extensions="{add-extension-extensions}"
+./gradlew addExtension --extensions='{add-extension-extensions}'
 ----
 endif::[]

